touchforms/backend/settings.py

```diff
@@ -19,6 +19,10 @@
 POSTGRES_TABLE = "formplayer_session"
 POSTGRES_JDBC_JAR = "%s/jrlib/postgresql-9.4-1201.jdbc41.jar"%os.path.dirname(os.path.abspath(__file__))
 
+# SQLite backend stuff
+USES_SQLITE = True
+SQLITE_TABLE = "formplayer_sqlstatus"
```

Comment:
nit: since this is something we probably never want overridden by localsettings we may want to put it in the persistence file or a constants file